crates/core/plugin_sm/src/plugin.rs

```diff
@@ -166,6 +205,7 @@ pub trait Plugin {
         module: &mut SoftwareModule,
         url: &DownloadInfo,
         logger: &mut BufWriter<File>,
+        _command_log: Option<&mut CommandLog>,
```

Comment:
I wasn't quite sure what's the convention to capture the generic logs emitted by functions like these in a workflow log, as they aren't really part of any script output.